example_code/activate_argcomplete_tryout_sh_autocomplete.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
NB: Files in example_code are just artifacts of my personal learning - ignore, no need to review these in detail. 

Doing so makes the changeset is quite small actually :)